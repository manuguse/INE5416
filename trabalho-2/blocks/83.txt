1 4 4 8 12 12 12 18 18 18
1 4 4 8 8 12 14 18 18 18
1 5 5 8 8 14 14 14 21 21
1 5 5 9 13 13 15 15 22 21
2 2 5 9 9 13 13 19 19 21
2 2 2 9 13 13 16 16 16 16
2 3 6 10 10 13 16 16 16 17
3 3 6 11 11 11 11 17 17 17
3 3 7 11 11 11 17 17 17 23
3 3 7 7 7 7 7 20 20 23