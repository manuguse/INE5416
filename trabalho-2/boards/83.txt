- 2 - - 2 - 3 - 5 4
- - 3 - 4 - - 2 - -
- 4 - - - - - - 3 -
- - 3 - 5 7 - - - 2
6 5 - - - 3 4 1 - 1
2 - - - - - 3 4 6 -
- 4 - - - - - - 5 -
7 - - - 5 4 - 4 - 5
- - - 6 - - 3 - - -
- - - 3 - 1 5 - 2 -