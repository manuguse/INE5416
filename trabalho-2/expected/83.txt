4 2 4 5 2 4 3 6 5 4
3 1 3 2 4 1 4 2 1 3
2 4 5 1 3 2 3 1 3 4
1 2 3 4 5 7 1 2 1 2
6 5 1 3 1 3 4 1 2 1
2 3 4 2 6 2 3 4 6 7
1 4 2 1 2 1 2 1 5 6
7 3 1 7 5 4 1 4 7 5
6 2 4 6 3 2 3 2 1 2
5 1 2 3 6 1 5 1 2 1